cdatransform/extract/pdc.py

```diff
@@ -123,6 +217,26 @@ def get_file_metadata(file_metadata_record) -> dict:
         ]
     }
 
+        
+def agg_cases_info_for_study(study,demo,diag,samp,added_info):
+    out = dict({})
+    out = []
+    for demo_case in demo:
+        case_id = demo_case['case_id']
+        demo_case.update(added_info)
+        for diag_ind in range(len(diag)):
+            if diag[diag_ind]['case_id'] == case_id:
+                demo_case['diagnoses'] = diag.pop(diag_ind)['diagnoses']
+                break
+        for samp_ind in range(len(samp)):
```

Comment:
I don't think saving a couple fo characters for samp_ind.   I'd rather see sample_idx or  sample_index or even just instance.

Reply:
Good idea. Will alter and push